### ISSUE TITLE
Log link to token incorrect.

### ISSUE THREAD

**Issue by lodici** [magarena/magarena#1055]
Identified in #1044 (may or may not be related), the "captain of the watch" log link is incorrect, highlighting a land card on the battlefield and displaying the land card if clicked (test using TestTokenCopy) -
![screenshot00567](https://cloud.githubusercontent.com/assets/2443519/22368873/b917f9f6-e480-11e6-8e98-49dda65bb724.jpg)

**Comment by ShawnieBoy** [comment]
It's not a link to the token, it's a link to the card that's been exiled (which still isn't linked to)

**Comment by lodici** [comment]
Yes, you are right. There is in fact no link to the token at all. Probably should change issue heading.
